libs/gi/page-home/src/QuickLinksCard.tsx

```diff
@@ -124,45 +104,69 @@ export default function QuickLinksCard() {
         avatar={<InsertLink fontSize="large" />}
       />
       <Divider />
-      <CardContent sx={{ display: 'flex', flexDirection: 'column', gap: 1 }}>
-        {buttons.map((btnProps, i) => {
-          const { title, icon, tooltip, color } = btnProps
-          let button
-          if ('to' in btnProps)
-            button = (
-              <Button
-                fullWidth
-                key={i}
-                color={color}
-                component={RouterLink}
-                to={btnProps.to}
-                startIcon={icon}
-              >
-                {title(t)}
-              </Button>
-            )
-          if ('url' in btnProps)
-            button = (
-              <Button
-                fullWidth
-                key={i}
-                color={color}
-                component={Link}
-                href={btnProps.url}
-                target="_blank"
-                rel="noopener"
-                startIcon={icon}
-              >
-                {title(t)}
-              </Button>
-            )
-          if (!button) return null
-          return (
-            <Tooltip key={i} title={tooltip(t)} placement="top" arrow>
-              {button}
-            </Tooltip>
-          )
-        })}
+      <CardContent
+        sx={{ display: 'grid', gridTemplateColumns: 'repeat(2,1fr)', gap: 1 }}
+      >
+        <CardThemed bgt="light">
+          <CardContent>
+            <Typography marginBottom={1} variant="h6">
+              Genshin Optimizer
+            </Typography>
+            <Box sx={{ display: 'flex', flexDirection: 'column', gap: 0.5 }}>
+              {genshin_optimizer_links.map((link) => {
+                const { title, icon, url } = link
+                return (
+                  <Box sx={{ display: 'flex', gap: 2 }}>
```

Comment:
Gap between icon and text is perhaps too big, I also don't like how the blue link color clash with the blue background. I think it may make sense to convert them into "buttons"
```suggestion
                  <CardActionArea sx={{ display: 'flex', gap: 1 }} component={Link} href={url} target="_blank" rel="noopener">
```

Reply:
![image](https://github.com/frzyc/genshin-optimizer/assets/25755821/61e286b6-58ed-48c4-a5e0-8b64e203d559)
 
 
 Amended to that, think it looks great! Thoughts?